scripts/deploy-nginx.sh

```
@@ -0,0 +1,34 @@
+        docker stop "$EXISTING_CONTAINER_ID" >/dev/null 2>&1 || true
+        docker rm "$EXISTING_CONTAINER_ID" >/dev/null
+    fi
+fi
+
+NGINX_FILES="-f docker/docker-compose.nginx.yml"
+
+echo "==> Stopping nginx..."
+docker compose $NGINX_FILES down --remove-orphans
+
+echo "==> Starting nginx..."
+docker compose $NGINX_FILES up -d
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)